Makefile

```diff
@@ -78,38 +79,23 @@ prepare-test-verifiables: clean
 	@cp scripts/prepare-test-verifiables.js .build/
 	@scripts/prepare_test_verifiables.sh
 
-.PHONY: check-openapi-specs
-check-openapi-specs: generate-openapi-spec
-
-.PHONY: generate-openapi-spec
-generate-openapi-spec: clean
-	@echo "Generating and validating controller API specifications using Open API"
-	@mkdir -p ${OPENAPI_SPEC_PATH}
-	@SPEC_META=$(VC_REST_PATH) SPEC_LOC=${OPENAPI_SPEC_PATH}  \
-	DOCKER_IMAGE=$(OPENAPI_DOCKER_IMG) DOCKER_IMAGE_VERSION=$(OPENAPI_DOCKER_IMG_VERSION)  \
-	scripts/generate-openapi-spec.sh
-
-.PHONY: generate-openapi-demo-specs
-generate-openapi-demo-specs: clean generate-openapi-spec vc-server-docker
-	@echo "Generate demo agent rest controller API specifications using Open API"
-	@SPEC_PATH=${OPENAPI_SPEC_PATH} OPENAPI_DEMO_PATH=test/bdd/fixtures/openapi-demo \
-    	DOCKER_IMAGE=$(OPENAPI_DOCKER_IMG) DOCKER_IMAGE_VERSION=$(OPENAPI_DOCKER_IMG_VERSION)  \
-    	scripts/generate-openapi-demo-specs.sh
-
-.PHONY: run-openapi-demo
-run-openapi-demo: generate-test-keys  generate-openapi-demo-specs
-	@echo "Starting demo vc rest containers ..."
-	@FIXTURES_PATH=test/bdd/fixtures  \
-        scripts/run-openapi-demo.sh
+.PHONY: open-api-spec
+open-api-spec: clean
+	@GOBIN=$(GOBIN_PATH) go install github.com/go-swagger/go-swagger/cmd/swagger@$(SWAGGER_VERSION)
+	@echo "Generating Open API spec"
+	@mkdir $(SWAGGER_DIR)
+	@$(GOBIN_PATH)/swagger generate spec -w ./cmd/vc-rest -x github.com/trustbloc/orb -o $(SWAGGER_OUTPUT)
```

Comment:
why we are referring trustbloc orb here?

Reply:
this excludes generating OpenAPI spec for Orb, and we have at least 2 reasons to do that: 1) spec for Orb is broken; 2) we don't need it in scope of VC services cc @sudeshrshetty